server/db/models/review.js

```diff
@@ -0,0 +1,25 @@
+var mongoose = require('mongoose');
+
+
+var ReviewSchema = new mongoose.Schema({
+  stars: {
```

Comment:
maybe this should be required? I feel like at least stars OR description have to be required, otherwise the review would have no content